flake.nix

```diff
@@ -171,7 +171,19 @@
               source ${writeScript ./nix/shell.sh}
             '';
           });
-
+        devShells.ktls = pkgs.mkShell {
```

Comment:
I wonder if this should be "minimal" instead of "ktls". Could we use this shell instead in other places where we just want to run the unit tests?

Reply:
updated